File: output/CodeLlama/avatar/Python/Java/atcoder_ABC119_B.java
Scanner sc = new Scanner(System.in);
int N = sc.nextInt();
double[] x = new double[N];
for (int i = 0; i < N; i++) {
    x[i] = sc.nextDouble();
}
double sum = 0;
for (int i = 0; i < N; i++) {
    sum += x[i];
}
System.out.println(sum);

File: output/CodeGen/avatar/Java/Python/atcoder_ABC119_B.py
def main(args):
    sc = Scanner(System.in)
    N = sc.nextInt()
    x = [0] * N
    u = []
    for i in range(N):
        x[i] = sc.nextDouble()
        u.append(sc.next())
    print(solve(x, u))
    sc.close()

def solve(x, u):
    return sum(map(lambda i: x[i] * (u[i] == "JPY" and 1 or 380000), range(len(x))))